src/main/resources/org/jenkinsci/plugins/reverse_proxy_auth/ReverseProxySecurityRealm/config.jelly

```diff
@@ -40,8 +40,7 @@ THE SOFTWARE.
       <f:textbox name="customLogOutUrl" value="${instance.customLogOutUrl}" />
     </f:entry>
 	<f:entry title="${%Server}" >
-	  <f:textbox name="server" value="${instance.server}"
-            checkUrl="'${rootURL}/securityRealms/LDAPSecurityRealm/serverCheck?field=server&amp;server='+encodeURIComponent(this.value)+'&amp;managerDN='+encodeURIComponent(this.form.elements['managerDN'].value)+'&amp;managerPassword='+encodeURIComponent(this.form.elements['managerPassword'].value)"/>
+	  <f:textbox name="server" value="${instance.server}"/>
```

Comment:
So does this break validation methods?
I only see https://github.com/jenkinsci/reverse-proxy-auth-plugin/blob/master/src/main/java/org/jenkinsci/plugins/reverse_proxy_auth/ReverseProxySecurityRealm.java#L744-L801 and it becomes unused after this change?

Reply:
Yes, as mentioned in the body above, it is a relic of older times. We will be re-adding validation at a later date, but it is not going to be part of this PR.

Reply:
Less like breaking, more like removing something that didn't work.

Reply:
Ack